tests/test_cli/test_main.py

```
@@ -0,0 +1,200 @@
+    assert "does not exist" in result.output
+
+def test_cli_invalid_config_format(cli_runner, tmp_path):
+    """Test CLI with invalid config format."""
+    config_path = tmp_path / "invalid.yaml"
+    config_path.write_text("invalid: yaml: content")
+    
+    result = cli_runner.invoke(main, ["--config-file", str(config_path)])
+    assert result.exit_code != 0
+    assert "Error:" in result.output
+
+def test_cli_valid_config(cli_runner, config_file, mocker):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_cli_valid_config`

Add return type annotation: `None`

(ANN201)